modules/component/src/index.ts

```diff
@@ -1,11 +1,8 @@
-export { Hook$ } from './hook$/hook$.decorator';
-export { selectChange } from './hook$/operators/selectChange';
-export { HostListener$ } from './host-listener$/host-listener$.decorator';
-export { Input$ } from './input$/input$.decorator';
-export { LocalStateService } from './local-state/local-state';
-export { selectSlice } from './local-state/operators/selectSlice';
-export { Push$Pipe } from './push$/push$.pipe';
-export { Async$Pipe } from './push$/async$.pipe';
-export { detectChanges } from './push$/operators/detectChanges';
-export { LetDirective } from './let/let.directive';
+export * from './core/operators/observeOncePerAnimationFrame';
```

Comment:
Can you revert this, and make it specific exports please?

For example.

```ts
export { PushPipe } from './push';
```

We only want to export the public stuff